package.json

```diff
@@ -249,7 +249,7 @@
     "tiny-invariant": "^1.2.0",
     "ua-parser-js": "^1.0.35",
     "use-resize-observer": "^9.1.0",
-    "uuid": "^8.3.2",
+    "uuid": "^9.0.0",
```

Comment:
i might not need to upgrade this actually, i thought we didn't have the package at all